src/components/badges/Badges.stories.mdx

```diff
@@ -53,35 +70,35 @@ Status Badges are used to indicate the status of a particular user and are usual
 or username
 
 export const canvasExampleArgs = {
-    ...defaultArgs,
+    ...mentionBadgeDefaultArgs,
 };
 
 <Canvas hidden>
     <Story
         name="Mention Badges"
-        argTypes={argTypes}
+        argTypes={mentionBadgeArgTypes}
         args={{
-            ...canvasExampleArgs,
+            ...mentionBadgeDefaultArgs,
         }}
     >
-        {Template.bind({})}
+        {MentionTemplate.bind({})}
     </Story>
     <Story
         name="Unread Badges"
-        argTypes={argTypes}
+        argTypes={unreadBadgeArgTypes}
         args={{
-            ...canvasExampleArgs,
+            ...unreadBadgeDefaultArgs,
         }}
     >
-        {Template.bind({})}
+        {UnreadTemplate.bind({})}
     </Story>
     <Story
         name="Status Badges"
```

Comment:
I am adding this in another PR, just kept it the same as mention in order to not break the story